tests/components/motionblinds_ble/test_button.py

```diff
@@ -0,0 +1,36 @@
+"""Tests for Motionblinds BLE buttons."""
+
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.components.button import DOMAIN as BUTTON_DOMAIN, SERVICE_PRESS
+from homeassistant.components.motionblinds_ble.const import (
+    ATTR_CONNECT,
+    ATTR_DISCONNECT,
+    ATTR_FAVORITE,
+)
+from homeassistant.const import ATTR_ENTITY_ID, Platform
+from homeassistant.core import HomeAssistant
+
+from . import setup_platform
+
+
+@pytest.mark.parametrize(("button"), [ATTR_CONNECT, ATTR_DISCONNECT, ATTR_FAVORITE])
+async def test_button(hass: HomeAssistant, button: str) -> None:
+    """Test states of the button."""
+
+    _, name = await setup_platform(hass, [Platform.BUTTON])
+
+    with patch(
+        f"homeassistant.components.motionblinds_ble.MotionDevice.{button}"
+    ) as command:
```

Comment:
I think having a centralized mock is of a great benefit here

Reply:
You propose setting a Mock on each instance of the button entity? How would you do that?

Reply:
No I mean a complete mock for `MotionDevice`. checkout `airgradient` for example. We create 1 mock for the client and we patch the methods that are needed and that;s it

Reply:
This has been solved!